webuild-drafts/IntermediaryTesting.md

```diff
@@ -0,0 +1,538 @@
+
+## 2.2       The RP Intermediary Role
+
+As defined in draft ETSI TR 119 479-1, an RP Intermediary is:
+
+“A party that provides a service to Relying Parties related to the handling of the interaction with wallets (a.o. EUDIWs) or other solutions for identification, authentication or attribute attestation of persons.”
+
+RP Intermediaries are also commonly referred to as “identity brokers” in existing electronic identification schemes. Their role is to abstract the complexity of wallet interactions, protocol implementations, and attestation validation, providing a unified interface for Relying Parties.
+
+# 3         Importance of Interemediaries to the EUDIW ecosystem
+
+Draft ETSI TR 119 479-1 identifies four key benefits that RP Intermediaries bring to the EUDIW ecosystem:
+
+## 3.1       Inclusion
+
+Small and medium enterprises (SMEs) and organizations without dedicated identity specialists will struggle to implement EUDIW integration independently. RP Intermediaries enable these organizations to participate in the digital identity ecosystem without requiring specialized in-house expertise.
+
+**Impact:** Without RP Intermediaries, a significant digital divide would emerge between organizations with substantial IT resources and those without, limiting the reach and effectiveness of the EUDIW initiative.
+
+## 3.2       Speed of Adoption
+
+The scarcity of identity and cybersecurity specialists creates a bottleneck for EUDIW implementation. Organizations already integrated with an RP Intermediary can begin using EUDIW capabilities immediately once their intermediary supports it, rather than queuing for specialist resources.
+
+**Impact:** RP Intermediaries break the “chicken and egg” problem by enabling rapid scaling of Relying Party adoption, which in turn increases wallet holder incentives to obtain and use EUDIWs.
+
+## 3.3       Cost Efficiency
+
+Building and maintaining interfaces and configurations for 27+ national EUDIW implementations represents substantial development and operational costs. RP Intermediaries amortize these costs across multiple Relying Parties.
+
+**Cost Analysis:**
+
+- **Direct integration**: Each RP must implement and maintain connections to all relevant wallet providers
+  
+- **Intermediated integration**: One-time integration with RPI, which maintains connections to all wallet providers
+  
+- **Economies of scale**: Shared infrastructure, expertise, and compliance costs
+  
+
+## 3.4       Security and Privacy
+
+Organizations lacking identity specialists may implement EUDIW integration with suboptimal security and privacy controls, especially when under regulatory deadline pressure. RP Intermediaries can provide:
+
+- Proven, audited implementation patterns
+  
+- State-of-the-art security controls
+  
+- Expert guidance and support
+  
+- Consistent privacy protection mechanisms
```

Comment:
I suppose that we may not try to sell to the users an IT externalization strategy through this document.